### DISCUSSION TITLE
Coinbase wallet not showing name and logo

### DISCUSSION THREAD

**Discussion by datznt** [wevm/wagmi#2191] [Q&A]
Hi, I using wagmi and coinbase wallet sdk. 
I tried to initialize 'CoinbaseWalletConnector' with options 'appName' but when the wallet connect screen comes up I see the app name is localhost.
I debug 'CoinbaseWalletSDK.js' on chrome devtools and noticed that 'walletExtension' flag is always valid so skip setting app info step.
How should I use it?

![image](https://user-images.githubusercontent.com/41461026/232460650-953e6b6e-8dbf-497b-be1f-ec7a0853c306.png)

![image](https://user-images.githubusercontent.com/41461026/232461997-3e29b933-df07-42a4-97ea-af89f46c241d.png)

**Comment by tmm** [comment]
Probably best to raise this on the Coinbase Wallet SDK https://github.com/coinbase/coinbase-wallet-sdk
